jcl/src/java.base/share/classes/com/ibm/jit/JITHelpers.java

```diff
@@ -69,6 +70,47 @@ private static JITHelpers jitHelpers() {
 		return helpers;
 	}
 
+	/*
```

Comment:
Can you add doxygen format comments? You can checkout the references of such comments on Various JCL code for example https://github.com/eclipse-openj9/openj9/blob/73dac1a16e721e8d62bf2ac6bec116e4ec4464c4/jcl/src/java.base/share/classes/java/lang/Class.java#L345-L353